docs/content/commands/npm-view.md

```diff
@@ -17,50 +17,48 @@ aliases: info, show, v
 This command shows data about a package and prints it to the stream
 referenced by the `outfd` config, which defaults to stdout.
 
-To show the package registry entry for the `connect` package, you can do
-this:
+As an example, to view information about the `connect` package from the registry, you would run:
 
 ```bash
 npm view connect
 ```
 
-The default version is "latest" if unspecified.
+The default version is `"latest"` if unspecified.
 
 Field names can be specified after the package descriptor.
 For example, to show the dependencies of the `ronn` package at version
-0.3.5, you could do the following:
+`0.3.5`, you could do the following:
 
 ```bash
 npm view ronn@0.3.5 dependencies
 ```
 
 You can view child fields by separating them with a period.
-To view the git repository URL for the latest version of npm, you could
-do this:
+To view the git repository URL for the latest version of `npm`, you would run the following command:
 
 ```bash
 npm view npm repository.url
 ```
 
 This makes it easy to view information about a dependency with a bit of
-shell scripting.  For example, to view all the data about the version of
-opts that ronn depends on, you can do this:
+shell scripting. For example, to view all the data about the version of
+`opts` that `ronn` depends on, you could write the following:
 
 ```bash
 npm view opts@$(npm view ronn dependencies.opts)
 ```
 
 For fields that are arrays, requesting a non-numeric field will return
-all of the values from the objects in the list.  For example, to get all
-the contributor names for the "express" project, you can do this:
+all of the values from the objects in the list. For example, to get all
+the contributor names for the `express` package, you would run:
 
 ```bash
 npm view express contributors.email
 ```
 
 You may also use numeric indices in square braces to specifically select
```

Comment:
```suggestion
You may also use numeric indices in square brackets to specifically select
```
? i realize the terms vary across the world but “square brackets” seems most universal in my experience.